components/ws-manager-mk2/main.go

```diff
@@ -156,8 +156,15 @@ func main() {
 
 	go func() {
 		for {
-			<-mgr.Elected()
-			activity.ManagerStartedAt = time.Now()
+			select {
+			case <-mgrCtx.Done():
+				return
+			case <-mgr.Elected():
```

Comment:
i saw that `Elected()` gets closed once elected, it probably doesn't make sense to try to read again from it in the for loop? Or do we exit when we lose election?

Reply:
> Or do we exit when we lose election?

I tried that approach, but that means the stand-by replica constantly restarts (never gets ready).
The issue here is that we baked the grpc server in the controller.

Reply:
Testing the last commit: if I change the lease using an invalid `holderIdentity`, after the election timeout, the current leader is restarted